frontend/src/pages/org/collections-new.ts

```diff
@@ -33,6 +36,7 @@ export class CollectionsNew extends LiteElement {
       <btrix-collection-editor
         .authState=${this.authState}
         orgId=${this.orgId}
+        .orgStorageQuotaReached=${this.orgStorageQuotaReached!}
```

Comment:
Use `?` to [prefix boolean attributes](https://lit.dev/docs/templates/expressions/#boolean-attribute-expressions):
```suggestion
        ?orgStorageQuotaReached=${this.orgStorageQuotaReached}
```